modules.d/95iscsi/iscsiroot.sh

```diff
@@ -229,7 +229,8 @@ handle_netroot() {
             echo "$target"
         done
     })
-    [ -z "$targets" ] && warn "Target discovery to $iscsi_target_ip:${iscsi_target_port:+$iscsi_target_port} failed with status $?" && return 1
+    ret=$?
```

Comment:
It should be also declared as `local`.

Reply:
Sure, you're right. I didn't know this file is supposed to be POSIX (and that posix doesn't know local). Prefixing it makes sense as well.